src/SSHDebugPS/UI/Controls/Automation/ContainerListBoxItemAutomationPeer.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Windows.Automation;
+using System.Windows.Automation.Peers;
+using System.Windows.Automation.Provider;
+
+namespace Microsoft.SSHDebugPS.UI
+{
+    /// <summary>
+    /// This automationpeer class is here to support the custom expand/collapse action on the listbox item
+    /// </summary>
+    /// <typeparam name="T">Must be IContainerViewModel</typeparam>
+    public class ContainerListBoxItemAutomationPeer<T> : 
+        ListBoxItemAutomationPeer, 
+        IExpandCollapseProvider 
+        where T : IContainerViewModel
+    {
+        public ContainerListBoxItemAutomationPeer(object item, ContainerListBoxAutomationPeer ownerAutomationPeer)
+            : base(item, ownerAutomationPeer)
+        { }
+
+        /// <summary>
+        /// Item is from the constructor and is the IContainerViewModel, not the listboxitem
+        /// </summary>
+        protected T ViewModel
+        {
+            get
+            {
+                return (T)Item;
+            }
+        }
+
+        #region IExpandCollapseProvider
+        public ExpandCollapseState ExpandCollapseState
+        {
+            get
+            {
+                if (ViewModel.IsExpanded)
```

Comment:
`return ViewModel.IsExpanded ? ExpandCollapseState.Expanded : ExpandCollapseState.Collapsed`

Reply:
that works too?

Reply:
You have different styles for the single line if statement. One with curly braces, the other without.